packages/typespec-ts/test/unit/apiVersion.spec.ts

```diff
@@ -174,15 +174,15 @@ const buildPathReturn_WithDefault = () => {
   
   /**
    * Initialize a new instance of \`testClient\`
-   * @param endpoint - The endpoint to use.
+   * @param endpointParam - The endpoint to use.
    * @param options - the parameter for all optional parameters
    */
   export default function createClient(
-    endpoint: string,
+    endpointParam: string,
```

Comment:
rename `endpoint` into `endpointParam` to avoid confusion with `options.endpoint`